lm_eval/__main__.py

```diff
@@ -210,9 +217,10 @@ def cli_evaluate(args: Union[argparse.Namespace, None] = None) -> None:
                         re.sub("/|=", "__", args.model_args), task_name
                     )
                     filename = path.joinpath(f"{output_name}.jsonl")
-
-                    with jsonlines.open(filename, "w") as f:
-                        f.write_all(samples[task_name])
+                    samples_dumped = json.dumps(
```

Comment:
Unless I am missing something, this is no longer jsonl but a serialized json list. Was this change intentional?

To get back the original behaviour we could do something like:
` lines = [json.dumps(
sample,
default=_handle_non_serializable,
ensure_ascii=False,
) for sample in samples[task_name]]`
`filename.write_text("\n".join(lines), encoding="utf-8")`